pkg/MoreLinq.Source.MoreEnumerable.Acquire.nuspec

```diff
@@ -1,27 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All source packages appear to have been and need to be restored because they are independent of issue #194.